clash-lib/prims/vhdl/Clash_Sized_Internal_Signed.primitives.yaml

```diff
@@ -75,10 +75,16 @@
     workInfo: Constant
 - BlackBox:
     name: Clash.Sized.Internal.Signed.*#
-    kind: Expression
+    kind: Declaration
     type: '(*#) :: KnownNat
       n => Signed n -> Signed n -> Signed n'
-    template: resize(~ARG[1] * ~ARG[2], ~LIT[0])
+    template: |-
+      ~GENSYM[~RESULT_mult][0] : block
+        signal ~GENSYM[~RESULT_mult_full][1] : std_logic_vector(~SIZE[~TYPO]*2-1 downto 0);
```

Comment:
Why the conversion through `std_logic_vector`? We are allowed to slice `signed`, no?

Reply:
That probably was leftover from my initial attempt to do this in a single expression.
I've removed the std_logic_vector now.